### ISSUE TITLE
Allow rating the reviews

### ISSUE THREAD

**Issue by pizzamaker** [eloquence/lib.reviews#64]
Amazon has a neat system for bringing to the fore the most useful critiques and burrowing the least useful, through the question "Was this review helpful to you?". I think a system like that would diminish the need for moderation of content. Also, buttons for reporting rule-breaking.

**Comment by eloquence** [comment]
Yeah, some kind of upvoting/downvoting may be helpful. But I want to be very careful with these systems, as they can easily introduce unintended biases. For example, if the system relies on # of votes, a new review will have a hard time rising to the top. This is a typical issue on StackExchange sites, where better answers that come in later are often hidden somewhere at the bottom.

Another issue to guard against is of course votespam/flagspam -- so in some ways you just end up shifting the moderation problem down a level, where it can become harder to see. There are sites that make usernames of raters/flaggers visible to an extent to guard against that.

Before implementing voting, I'd like to try putting more heft behind the "teams" feature by making it possible to your review to a team you're part of, following teams, and then ranking reviews according to teams you follow.

**Comment by eloquence** [comment]
from relevant IRC discussion:

**WinterFox[m]18:** Steam has a really good review section despite having a large userbase of kids 
They used to have a lot of memes in the reviews but that got stopped mostly by adding a funny button next to like/dislike

https://matrix.org/_matrix/media/v1/download/matrix.org/vvJvnzsswHaUvePjIoxhtcrG

For more popular games you will see 7 paragraph reviews breaking down every part of the game

**Eloquence:** I do like "x of y found helpful", and it can be more easily adjusted to not suffer from age bias in the same way cumulative upvotes do

still gameable, especially once we fully open up registration

being very transparent about why/how something is ranked higher is very important, I think -- I want anyone to be able to look at the data and help identify suspicious patterns

some of you may remember kuro5hin, which had a score-rating feature that let users rate comments 1-5 and would show you the average, together with the list of users who rated

trusted users were able to also rate 0 to hide a comment, which would factor into the average -- once it falls below 1, it gets hidden

**Comment by nortix** [comment]
Why not just insert the Upvote Button? If you do not find the review helpful you can leave out your vote.
